.github/workflows/build-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: build-tests
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  pylint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint sphinx sphinx_rtd_theme
+          pip install -r requirements.txt
+      - name: Run pylint
+        run: |
+          pylint --disable=I --disable=W --disable=C --disable=R p5
+
+  pytest:
+    needs : pylint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install pytest
+          sudo apt-get install python-opengl libglfw3 -y
```

Comment:
This has to be installed first. See the existing GHAs and keep them the same order. That's why the actions are failing as well.

Reply:
The order is the same as existing GHAs.The `python-opengl` isn't installing in the ubuntu image for some reason.I even tried installing the package through a seperate GHA on another repo yet it failed , seems like an issue with the apt-package itself.Can you re-run the pytest action on the latest commit and tell if the same issue persists there as well?

Reply:
Hey, I just checked can you change it to `python3-opengl` from `python-opengl` and see if it works? I think the name has changed in newer version. https://packages.ubuntu.com/jammy/python/python3-opengl

Reply:
Ya that was the issue.Now it's working fine.